### PR TITLE
Streams 2 migration

### PR DESCRIPTION
**Pull request by jasonmartens**
@jiahuang please review.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -8,8 +8,10 @@ organization := "io.dronekit"
 
 resolvers += "Artifactory" at "https://dronekit.artifactoryonline.com/dronekit/libs-snapshot-local/"
 
-credentials += Credentials("Artifactory Realm", "dronekit.artifactoryonline.com", "publish", "Km4-PSH-aEM-6Fm")
+credentials += Credentials(Path.userHome / ".sbt" / ".credentials")
```
Comment (jiahuang) [inline build.sbt:11]: :star2:

File: src/main/resources/application.conf [hunk 2]
```diff
@@ -1,5 +1,9 @@
 akka {
   loglevel = DEBUG
+  http {
```
Comment (jiahuang) [inline src/main/resources/application.conf]: should this be a reference.conf?

File: build.sbt (hunk 1 above)
Comment (jasonmartens) [inline build.sbt:11]: Yeah, I already deleted those credentials.

File: src/test/scala/TestClient.scala [hunk 3]
```diff
@@ -21,13 +22,15 @@ object TestClient extends App {
   val logger = Logging(system, getClass)
 
 
-  val file = new File("/Users/jiahuang/3dr/stream-s3-uploader/src/test/resources/recap_res.obj.zip")
+  val file = new File("/Users/jasonmartens/Downloads/Star wars.mp4")
```
Comment (jiahuang) [inline src/test/scala/TestClient.scala:25]: ehhhhh can we have the test url be something not local? Maybe have it download a large file from the 3dr.public bucket

Comment (jiahuang) [inline src/test/scala/TestClient.scala:25]: I guess that means you also have to add in downloading into the test script. Whatever, maybe do that later.

**Pull request merged**